tests/snapshots/run_file__log.hvm.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_file/log.hvm
+---
+"Hi"
```

Comment:
We should probably show both the output and the return of the program in these tests, but maybe we do that in another PR (one for adding these IO tests).

Reply:
agreed